packages/table/src/spectrum-table-head-cell.css

```diff
@@ -0,0 +1,339 @@
+}
+:host([dir='rtl']) {
+    text-align: right; /* [dir=rtl] .spectrum-Table-headCell */
+}
+:host([dir='ltr']) {
+    padding-left: var(
+        --spectrum-table-regular-header-padding-left
+    ); /* [dir=ltr] .spectrum-Table-headCell */
+    padding-right: var(--spectrum-table-regular-header-padding-right);
+}
+:host([dir='rtl']) {
+    padding-left: var(--spectrum-table-regular-header-padding-right);
+    padding-right: var(
+        --spectrum-table-regular-header-padding-left
+    ); /* [dir=rtl] .spectrum-Table-headCell */
+}
+:host {
+    border-radius: var(--spectrum-table-regular-header-border-radius);
+    box-sizing: border-box; /* .spectrum-Table-headCell */
+    cursor: default;
+    font-size: var(--spectrum-table-regular-header-text-size);
+    font-weight: var(--spectrum-table-regular-header-text-font-weight);
+    letter-spacing: var(--spectrum-table-regular-header-text-letter-spacing);
+    line-height: var(--spectrum-table-regular-header-text-line-height);
+    min-height: var(--spectrum-table-regular-header-min-height);
+    outline: 0;
+    padding-bottom: var(--spectrum-table-regular-header-padding-bottom);
+    padding-top: var(--spectrum-table-regular-header-padding-top);
+    text-transform: uppercase;
+    transition: color var(--spectrum-global-animation-duration-100, 0.13s)
+        ease-in-out;
+}
+:host([sortable]) {
+    cursor: pointer; /* .spectrum-Table-headCell.is-sortable */
+}
+:host([sort-direction='asc']) .sortedIcon,
+:host([sort-direction='desc']) .sortedIcon {
+    display: inline-block; /* .spectrum-Table-headCell.is-sorted-asc .spectrum-Table-sortedIcon,
+   * .spectrum-Table-headCell.is-sorted-desc .spectrum-Table-sortedIcon */
+    margin-top: calc(var(--spectrum-global-dimension-size-25) * -1);
+}
+:host([sort-direction='asc']) .sortedIcon {
+    transform: rotate(
+        -90deg
+    ); /* .spectrum-Table-headCell.is-sorted-asc .spectrum-Table-sortedIcon */
+}
+:host {
+    position: relative; /* .spectrum-Table-cell,
+   * .spectrum-Table-headCell */
+}
+:host(.focus-visible),
+:host([focused]) {
+    outline: none; /* .spectrum-Table-cell.focus-ring,
+   * .spectrum-Table-cell.is-focused,
+   * .spectrum-Table-headCell.focus-ring,
+   * .spectrum-Table-headCell.is-focused */
+}
+:host(:focus-visible),
+:host([focused]) {
+    outline: none; /* .spectrum-Table-cell.focus-ring,
+   * .spectrum-Table-cell.is-focused,
+   * .spectrum-Table-headCell.focus-ring,
+   * .spectrum-Table-headCell.is-focused */
+}
+:host([dir='ltr']) .spectrum-Table-headCell.focus-visible:before,
+:host([dir='ltr'][focused]) .spectrum-Table-headCell:before {
+    right: 0; /* [dir=ltr] .spectrum-Table-cell.focus-ring:before,
+   * [dir=ltr]  .spectrum-Table-cell.is-focused::before,
+   * [dir=ltr]  .spectrum-Table-headCell.focus-ring::before,
+   * [dir=ltr]  .spectrum-Table-headCell.is-focused::before */
+}
+:host([dir='ltr']) .spectrum-Table-headCell:focus-visible:before,
```

Comment:
Possibly missing something here, too.

Reply:
Looks like some of the origin selectors have two spaces `  ` between words which might mess up some of the rules in `spectrum-config.js`.